### PR TITLE
Update UNIX paths of config,db,logs, and sites

### PR DESCRIPTION
**Pull request by sagarnasit**
Closes: https://github.com/EasyEngine/easyengine/issues/1233

Depends on:
https://github.com/EasyEngine/config-command/pull/6
https://github.com/EasyEngine/auth-command/pull/13
https://github.com/EasyEngine/admin-tools-command/pull/18
https://github.com/EasyEngine/cron-command/pull/21
https://github.com/EasyEngine/site-command/pull/121
https://github.com/EasyEngine/site-type-wp/pull/54
https://github.com/EasyEngine/service-command/pull/11
https://github.com/EasyEngine/installer/pull/12

### CODE REVIEW

File: features/bootstrap/FeatureContext.php [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
 define( 'EE', true );
 define( 'EE_VERSION', trim( file_get_contents( EE_ROOT . '/VERSION' ) ) );
-define( 'EE_CONF_ROOT', '/opt/easyengine' );
+define( 'EE_ROOT_DIR', '/opt/easyengine' );
```
Comment (kirtangajjar) [inline features/bootstrap/FeatureContext.php]: BTW `EE_CONF_ROOT` is used in almost all repos(ee core, cron, admin-tools, service, auth, site-type-wp, config etc...). So if you're renaming it, just ensure that it is done in all the repos. If you haven't raised PR in these repos raise them and then post them as comment here.

**Comment by kirtangajjar** [comment]
Also I think we need to change ee cache dir. Currently I think it is `~/.ee/cache`. This needs discussion.

File: features/bootstrap/FeatureContext.php (hunk 1 above)
Comment (kirtangajjar) [reply features/bootstrap/FeatureContext.php]: @sagarnasit Make changes in other repos when you have time

**Comment by mbtamuli** [comment]
@sagarnasit Tests are failing because site root is defined wrong in tests. Example - 
https://github.com/EasyEngine/site-command/blob/3061753dd4baae83e5ffd70d23bcf0b2cc72c033/features/bootstrap/FeatureContext.php#L53
You need to make changes in all site types.

**Pull request merged**